helm/values.yaml

```diff
@@ -216,1462 +217,1489 @@ smtp:
-      limits:
-        cpu: 500m
-        memory: 1024Mi
-      requests:
-        cpu: 25m
-        memory: 256Mi
-  coordinating:
-    replicas: 2
-    resources:
-      limits:
-        cpu: 500m
-        memory: 1024Mi
-      requests:
-        cpu: 25m
-        memory: 256Mi
-  data:
-    replicas: 2
-    heapSize: 512m
-    resources:
-      limits:
-        cpu: 1
-        memory: 2048Mi
-      requests:
-        cpu: 50m
-        memory: 1024Mi
-    persistence:
-      size: 20Gi
+    enabled: false
+    name: "elasticsearch"
+    fullnameOverride: "graviteeio-apim-elasticsearch"
+    image:
+        repository: "bitnami/elasticsearch"
+        tag: "8.8.2"
+    master:
+        replicas: 2
+        persistence:
+            size: 4Gi
+        resources:
+            limits:
+                cpu: 500m
+                memory: 1024Mi
+            requests:
+                cpu: 25m
+                memory: 256Mi
+    coordinating:
+        replicas: 2
+        resources:
+            limits:
+                cpu: 500m
+                memory: 1024Mi
+            requests:
+                cpu: 25m
+                memory: 256Mi
+    data:
+        replicas: 2
+        heapSize: 512m
+        resources:
+            limits:
+                cpu: 1
+                memory: 2048Mi
+            requests:
+                cpu: 50m
+                memory: 1024Mi
+        persistence:
+            size: 20Gi
 
 alerts:
-  enabled: false
-  endpoints:
-    - http://localhost:8072/
-  security:
     enabled: false
-    username: admin
-    password: adminadmin
+    endpoints:
+        - http://localhost:8072/
+    security:
+        enabled: false
+        username: admin
+        password: adminadmin
 
 management:
-  type: mongodb
-  #url: your_public_ui_url
+    type: mongodb
+    #url: your_public_ui_url
 
 ratelimit:
-  type: mongodb
+    type: mongodb
 
 # Support for Gravitee.io Cockpit (cockpit.gravitee.io)
 cockpit:
-  enabled: false
-  keystore:
-    value: "base64 encoded value of the keystore provided by Cockpit (required)"
-    password:
-      #value: "keystores password provided by Cockpit"
-      #valueFrom:
-        #secretKeyRef:
-        #configMapKeyRef:
-  #truststore:
+    enabled: false
+    keystore:
+        value: "base64 encoded value of the keystore provided by Cockpit (required)"
+        password:
+            #value: "keystores password provided by Cockpit"
+            #valueFrom:
+            #secretKeyRef:
+            #configMapKeyRef:
+    #truststore:
     #value: base64 encoded value of the truststore provided by Cockpit (optional)
     #password:
-      #value: "truststore password provided by Cockpit"
-      #valueFrom:
-        #secretKeyRef:
-        #configMapKeyRef:
-  url: https://cockpit.gravitee.io
-  controller: https://cockpit-controller.gravitee.io
-  ssl:
-    verifyHostname: true
+    #value: "truststore password provided by Cockpit"
+    #valueFrom:
+    #secretKeyRef:
+    #configMapKeyRef:
```

Comment:
The rewrite of indentation implies that this comment looze their indentation.
This implies that when we will uncomment it, we may failed the config because of the missing indentation.